### PR TITLE
Add stats to RemoteTask

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
         <air.java.version>1.8.0-40</air.java.version>
 
         <dep.antlr.version>4.5.1</dep.antlr.version>
-        <dep.airlift.version>0.125</dep.airlift.version>
+        <dep.airlift.version>0.126-SNAPSHOT</dep.airlift.version>
```
Comment (nileema) [inline pom.xml]: This depends on https://github.com/airlift/airlift/pull/396 

If this seems reasonable, we can release airlift and update this dependency to 126

File: presto-main/src/main/java/com/facebook/presto/server/RemoteTaskStats.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import io.airlift.stats.DistributionStat;
+import org.weakref.jmx.Managed;
+import org.weakref.jmx.Nested;
+
+public class RemoteTaskStats
+{
+    private final DistributionStat updateRoundTripMillis = new DistributionStat();
+    private final DistributionStat infoRoundTripMillis = new DistributionStat();
+    private final DistributionStat responseSizeBytes = new DistributionStat();
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/server/RemoteTaskStats.java]: name this `updateResponseSizeBytes` and maybe add an `infoResponseSizeBytes`

Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/server/RemoteTaskStats.java]: Why is there no getter for response size?

Comment (nileema) [inline presto-main/src/main/java/com/facebook/presto/server/RemoteTaskStats.java]: The response for both update and info are the `TaskInfo` object. So I have only one. In the new code, I will have two stats, that will have a `TaskInfo` and `TaskStatus`.

**Comment by nileema** [comment]
@dain updated

**Comment by dain** [comment]
looks good

**Pull request merged**